calico-enterprise/operations/nftables.mdx

```diff
@@ -0,0 +1,206 @@
+---
+description: Install Calico Enterprise using the nftables data plane.
+title: "nftables"
+---
+
+# Install Calico Enterprise using the nftables data plane
+
+## Big picture
+
+Install $[prodname] using the nftables data plane.
+
+## Value
+
+Kubernetes introduced a beta status [kube-proxy Service implementation based on nftables](https://kubernetes.io/docs/reference/networking/virtual-ips/#proxy-mode-nftables)
```

Comment:
nftables kube-proxy is now GA and enabled by default in v1.33.

Reply:
I've added material to address this, thanks.